File: cml-compiler/cml-generator/src/test/resources/templates/lang/py/association/employee.py
class Employee:

    _employment = _Employment()

    def __init__(self, name: 'str', employer: 'Organization') -> 'None':
        self.__name = name
        self.__employer = employer

    @property
    def name(self) -> 'str':
        return self.__name

    @property
    def employer(self) -> 'Organization':
        return self.__employer

    def __str__(self) -> 'str':
        return "%s(name=%s, employer=%s)" % (
            type(self).__name__,
            self.name,
            self.employer
        )

File: cml-compiler/cml-generator/src/test/resources/templates/lang/java/association/employee.java
class Employee
{
    private static Employment employment;

    private final String name;
    private final Organization employer;

    public Employee(String name, Organization employer)
    {
        this.name = name;
        this.employer = employer;
    }

    public String getName()
    {
        return this.name;
    }

    public Organization getEmployer()
    {
        return this.employer;
    }

    public String toString()
    {
        return new StringBuilder(Employee.class.getSimpleName())
                   .append('(')
                   .append("name=").append(String.format("\"%s\"", getName())).append(", ")
                   .append("employer=").append(String.format("\"%s\"", getEmployer()))
                   .append(')')
                   .toString();
    }

    static void setEmployment(Employment association)
    {
        employment = association;
    }
}